### ISSUE TITLE
[QUESTION] Why SOAGM (aerosol phase methylglyoxal)  be removed from 12.8 in GEOSChem?

### ISSUE THREAD

**Issue by Lianheng257** [geoschem/geos-chem#568]
# Ask a question about GEOS-Chem:

Why SOAGM (aerosol phase methylglyoxal)  be removed from 12.8 in GEOSChem? I had checked the aerosol phase methylglyoxal is added in to SOAGX (aerosol phase glyoxal)? why do that? I want to get the reason that deletes the SOAGM and add both aerosol phase glyoxal and methylglyoxal into SOAGX?

**Comment by msulprizio** [comment]
Yumin Li also asked:

>In the wiki of GC species list there are still have the species: MGLY (gas-phase methylglyoxal) and  SOAGM (aerosol phase methylglyoxal). However, in GC 12.8 and GC 12.9 the SOAGM is not included in species list. There are still have the process of gas to aerosol transfer of methylgloxal (in KPP het.mod) but the aerosol phase methylgloxal be directly added into SOAGX (aerosol phase gloxal). And after that, the SOAGM also never appeared again in the GeosCore. I can not understand why mix the aerosol phase gloxal and and methylgloxal together in GC 12.8 and GC 12.9 because they have totally different molar mass and I also do not find any explain about this change.
>
>The gas to aerosol transfer of methylgoxal in GC 12.7:
![image](https://user-images.githubusercontent.com/4805437/101924565-d9ce9c80-3b9e-11eb-9e93-05a9dafee77f.png)
>The gas to aerosol transfer of methylgloxal (be added in aerosol phase glyoxal) in GC 12.9:
![image](https://user-images.githubusercontent.com/4805437/101924632-e652f500-3b9e-11eb-9943-ce5ee06eac0c.png)
>Many thanks for your help.

**Comment by msulprizio** [comment]
This change came in with the updated isoprene chemistry implemented in GEOS-Chem 12.8.0 following [Bates et al. (2019)](https://www.atmos-chem-phys.net/19/9613/2019/). We would refer you to that paper for the details. Please also see https://github.com/geoschem/geos-chem/pull/142.

**Comment by Lianheng257** [comment]
@msulprizio 
Thanks msulprizio!
I had read the Bates et al, (2019). Bates 2019 Shows that the production of methylglyoxal is much higher than that of glyoxal (20% molar yield from isoprene globally), but due to its low SOA yield, it contributes only 0.01 Tg iSOAa−1. So that the SOAGM be removed due to its low contribution? But why the process of gas-phase to aerosol-phase methylglyoxal process is still in KPP standard and the contribution from methylglyoxal be add in SOAGX?

**Comment by msulprizio** [comment]
Since this is more of a science question than technical question, I would defer to @kelvinhb.

**Comment by kelvinhb** [comment]
Yes, the individual tracer was removed just because of its low contribution, but we didn't want to artificially remove that SOA mass, so it's lumped in with the glyoxal SOA now. This fits with most of our other SOA tracers in the complex scheme, which aren't divided by individual species but rather by the mechanism through which they're formed -- e.g. IEPOX SOA includes SOA from other epoxides, isoprene nitrate SOA from all isoprene-derived nitrates, and LVOCOA being low-volatility highly-oxidized SOA from a variety of pathways. Since glyoxal and methylglyoxal SOA are formed through the same general uptake mechanism and should have the same chemical dependencies, and their individual contributions (especially of methylglyoxal) are relatively low, it made sense to combine them. They do have different molar masses, but our general scheme is already bad at tracking molar mass (e.g. doesn't account for particle-phase reactions following precursor uptake, and lumps other species with very different molar masses together into the other complex SOA categories), and we figured since glyoxal dominates it would not incur much bias to combine them. If people really want to separate them back out, though, we can do that!
